src/components/views/upload-keys.tsx

```diff
@@ -41,6 +42,11 @@ export const UploadKeys = ({navigation}) => {
   const [validationError, setValidationError] = useState<string>('');
   const [uploadToken, setUploadToken] = useState('');
   const [symptomDate, setSymptomDate] = useState('');
+  const [ref] = useFocusRef({
+    accessibilityFocus: true,
```

Comment:
`{timeout: 1000}` is enough. All the others are defaulted. Can I ask how did you choose 1000?

Reply:
It was already being used in a few other places so I assumed it was a safe bet